internal/handler/site/site_handler.go

```diff
@@ -71,6 +72,7 @@ func (h *SiteHandler) serveFile(w http.ResponseWriter, r *http.Request) {
 	if info.ContentType != "" {
 		w.Header().Set("Content-Type", info.ContentType)
 	}
+    w.Header().Set("Cache-Control", "public, max-age=31536000, no-cache")
```

Comment:
As seen in the referenced page, the `Cache-Control` header is used with `ETag` header, so it makes more sense to set it together with the `ETag` header.